package.json

```diff
@@ -7,10 +7,11 @@
     "format": "biome format ./",
     "lint:fix": "biome lint --fix ./",
     "lint": "biome lint ./",
+    "fix-commit": "node --run format:fix; node --run lint:fix",
     "pre-commit": "node --run lint:fix; node --run format:fix; node --run test:types; node --run test:unit; node --run test:e2e",
-    "test:e2e": "node --no-warnings --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=./coverage.lcov --test-reporter=spec --test-reporter-destination=stdout --import './build/snapshots.ts' --test --test-coverage-include='recipes/**/*' --test-coverage-exclude='**/*.e2e.{mjs,mts}' './packages/*/*.e2e.{mjs,mts}'",
+    "test:e2e": "NODE_OPTIONS='--no-warnings --experimental-test-coverage --test-reporter=spec --test-reporter-destination=stdout --import=\"./build/snapshots.mts\" --test-coverage-include=\"recipes/**/*\" --test-coverage-exclude=\"**/*.e2e.{mjs,mts}\"' npm run test:e2e --workspaces",
```

Comment:
Node doesn't allow these within `NODE_OPTIONS` (I don't know why—other experimental flags are supported):

* `--experimental-test-coverage`
* `--experimental-test-module-mocks`
* `--experimental-test-snapshots`

Can anyone think of another way to get these passed to the sub-process `npm run` will create?

Reply:
https://github.com/nodejs/node/pull/56114

Reply:
🤦‍♂️ there is a much easier way: `npm` supports passing along CLI args by appending `-- --FLAG_HERE`